package.json

```diff
@@ -4,7 +4,9 @@
   "license": "MIT",
   "scripts": {
     "build": "webpack",
-    "cypress": "cypress open"
+    "cypress": "cypress open",
+    "test-sirius": "cd internal/sirius && go test",
```

Comment:
Do we need these when it is simply `go test ./internal/sirius` or `go test ./internal/server` (or `go test ./...` to run all)?

Reply:
done